### PR TITLE
Update TP logging / lsp4j.jsonrpc jars

### PR DESCRIPTION
**Pull request by robstryker**

### CODE REVIEW

**Comment by robstryker** [comment]
testPR

**Comment by odockal** [comment]
testPR

File: targetplatform/rsp-target.target [hunk 1]
```diff
@@ -1,76 +1,90 @@
...
+
+			<!-- Docker Tooling deps -->
+			<unit id="org.assertj" version="1.7.1.v20170413-2026" />
+			<unit id="org.json" version="1.0.0.v201011060100" />
+			<unit id="org.mockito" version="1.9.5.v201605172210" />
+		</location>
+
+
+		<location includeAllPlatforms="false" includeMode="slicer" includeSource="true" type="InstallableUnit">
+			<repository location="http://oxbeef.net/permanent/p2/20191022/repository/" />
```
Comment (jeffmaury) [inline targetplatform/rsp-target.target]: We should first clone this repo to d.j.o in order to get reliable

Comment (sbouchet) [reply targetplatform/rsp-target.target]: I agree.

**Comment by odockal** [comment]
@robstryker  It is super broken. Yay! So we see that there is obviously something wrong, right?

**Comment by robstryker** [comment]
testPR

**Comment by robstryker** [comment]
testPR

**Comment by robstryker** [comment]
So the issue here was that we weren't reading the output streams of the rsp. They eventually filled up. My original fixes attempted to limit the logging level to something much lower, hoping that the channels wouldn't fill up. But this was insufficient. Seems the only solution is to make the itests actually read the streams and dump / ignore the values in them just so that the streams don't fill up. 

Build is green. I think the new jars work now.

File: targetplatform/rsp-target.target (hunk 1 above)
Comment (sbouchet) [reply targetplatform/rsp-target.target]: you can now use : https://download.jboss.org/jbosstools/updates/requirements/logback/1.2.3/

**Pull request merged**